speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/ConfigEnums.h

```diff
@@ -60,11 +60,14 @@ typedef unsigned int DataStorageType;
 #define DATA_STORAGE_TYPE_SQL       1
 
 /// @brief The different types of data that can be stored
+// TODO: update these to the newer database structure in the front-end.
+//  Current uses in the back-end for each value are listed below.
+//  Changing these without updating the front-end as well may break checkboxes and such
 typedef struct DataToStore
 {
-    bool EnvironmentData;
-    bool CarData;
-    bool HumanData;
-    bool InterventionData;
-    bool MetaData;
+    bool EnvironmentData; // Used as Gamestate
+    bool CarData; // Unused
+    bool HumanData; // Used as UserInput
+    bool InterventionData; // Used as Decisions
+    bool MetaData; // Unused
```

Comment:
Can we get this fixed in this PR as well? It's confusing to leave it in like this

Reply:
Not in this PR; fixing this would require front-end updates, which I don't know enough about yet. I can figure this out, but that would require further delaying this PR. Should this be added as a high-priority bug?

Reply:
Comments have been moved to SQLDatabaseStorage now to properly mark where changes are required